docs/en/02_Developer_Guides/02_Controllers/02_Routing.md

```diff
@@ -282,7 +282,7 @@ class BreadAPIController extends Controller
 ```
 
 <div class="alert" markdown="1">
-In SilverStripe Framework versions prior to 4.6, an empty key (`''`) must be used in place of the `'/'` key. When specifying an HTTP method, the empty string must be separated from the method (e.g. `'GET '`). The empty key and slash key are also equivalent in Director rules.
+In Silverstripe CMS Framework versions prior to 4.6, an empty key (`''`) must be used in place of the `'/'` key. When specifying an HTTP method, the empty string must be separated from the method (e.g. `'GET '`). The empty key and slash key are also equivalent in Director rules.
```

Comment:
Given Bryn's response on a previous comment, it is probably best to remove the term `Framework` from this sentence.

Reply:
Removed Framework from Silverstripe CMS Framework (3 files)

Reply:
Removed Framework from Silverstripe CMS Framework

Reply:
Silver**S**tripe CMS changed to Silverstripe CMS (Framework previously removed)